winpr/libwinpr/clipboard/posix.c

```diff
@@ -629,6 +659,7 @@ static void* convert_filedescriptors_to_file_list(wClipboard* clipboard, UINT32
 			size_t curLen = _wcsnlen(descriptors[x].cFileName, ARRAYSIZE(descriptors[x].cFileName));
 			alloc += WideCharToMultiByte(CP_UTF8, 0, descriptors[x].cFileName, (int)curLen, NULL, 0,
 			                             NULL, NULL);
+			alloc += count_special_chars(descriptors[x].cFileName) * 2;
```

Comment:
I guess this warrants a comment as to why 2, not 3, or something else.

(Each special character is escaped as `%FF` – 3 characters – but one of them is already included in `cFileName` length, previously added.)